src/content/updates/migration/developers.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Migration
+tabs: ['Overview', 'Designers', 'Developers']
+---
+
+<AnchorLinks>
+
+- [Repo architecture](#repo-architecture)
+- [Migration tooling](#migration-tooling)
+- [Migration strategy](#migration-strategy)
+- [Carbon components](#carbon-components)
+- [Carbon elements](#carbon-elements)
+- [Feedback](#feedback)
+
+</AnchorLinks>
+
+## Repo architecture
+
+Two of Carbon's previous repos will be deprecated with v10: carbon-icons and carbon-themes. Iconography and themes in v10 are now handled as packages in the carbon-elements repo. 
+
+[@carbon/colors](https://github.com/IBM/carbon-elements/tree/master/packages/colors)
+
+[@carbon/themes](https://github.com/IBM/carbon-elements/tree/master/packages/themes)
+
+## Migration tooling
+
+**Coming soon!** [carbon-upgrade](https://github.com/carbon-design-system/carbon-upgrade/blob/master/README.md) is a new tool to help developers upgrade to v10 through the command line. This tool is still experimental, and if you would like to contribute, the Carbon team would welcome the help! [Please leave ideas for migration tooling here](https://github.com/carbon-design-system/carbon-upgrade/issues/7). 
+
+## Migration strategy
```

Comment:
Product Team: Job to be done: Migrate our product from Carbon v9 to Carbon v10.

Be clear this is about migrating a product here, and it's not solely a dev task as new design artifacts may be required.

Reply:
Done.